src/utils/__tests__/Logger.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { Logger } from "../Logger";
+
+describe("Some logging behavior", () => {
+  const log = console.log; // save original console.log function
+  const warn = console.warn;
+  const debug = console.debug;
+  const error = console.error;
+  const info = console.info;
+  const logger = new Logger();
+  const message = "something";
+  logger.doActive = true;
+
+  beforeEach(() => {
+    console.log = jest.fn(); // create a new mock function for each test
+    console.warn = jest.fn();
+    console.error = jest.fn();
+    console.debug = jest.fn();
+    console.info = jest.fn();
+  });
+  afterAll(() => {
```

Comment:
Consider switching this to `afterEach` and including the `logger.doActive` assignments in these lifecycle hooks. For example

```ts
beforeEach(() => {
  logger.doActive = true;
});

afterEach(() => {
  logger.doActive = false;
});
```
This way you don't have to worry if a test changes it (no log in production) and forgets to reset for the next test.